### PR TITLE
PS-5659-RHEL-requires-mysql-mod-disabled-for-packages-installation ad…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ded RHEL to the supported releases and added a note in the installing PS using downloaded rpm packages - section 2 before the installation step.

### CODE REVIEW

File: doc/source/installation/yum_repo.rst [hunk 1]
```diff
@@ -189,13 +190,21 @@ Installing |Percona Server| using downloaded rpm packages
 
 	 percona-server-80-debuginfo-8.0.13-3.el7.x86_64.rpm
 	 percona-server-client-80-8.0.13-3.el7.x86_64.rpm
-	 percona-server-devel-80-8.0.13-3.el7.x86_64.rpm
+	 percona-server-devel-80-8.0.13-3.el7.x86_64.rpm6
```
Comment (hrvojem) [inline doc/source/installation/yum_repo.rst]: where did the `6` came from :)

**Comment by laurynas-biveinis** [comment]
Make sure to squash commits on merge

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
The fix branches must be GCA-based

**Comment by patrickbirch** [comment]
Updated branch to GCA

**Comment by laurynas-biveinis** [comment]
This cannot possibly be GCA-based: there are three commits, none of them is a merge from a lower version

**Comment by patrickbirch** [comment]
Thank you. Yes, this is the one for version 8. I am waiting for approval of the 5.7 version before fixing this version and updating it. 
When I created this PR I was unaware that I needed the merges from the lower version. Hrvoje and George have explained that I should do the lowest version first, have that one approved, before working on the next version.

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by laurynas-biveinis** [comment]
@patrickbirch this is still not GCA. Was it merged by mistake or did you think it was actually GCA?

**Comment by patrickbirch** [comment]
My mistake. Will fix.

**Pull request merged**